controllers/reconcilers/kserve_istio_peerauthentication_reconciler.go

```diff
@@ -52,24 +52,7 @@ func NewKServeIstioPeerAuthenticationReconciler(client client.Client) *KserveIst
 }
 
 func (r *KserveIstioPeerAuthenticationReconciler) Reconcile(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
-	log.V(1).Info("Reconciling PeerAuthentication for target namespace")
```

Comment:
If we remove the entire reconciliation process, that won't delete the existing resources for clusters doing an upgrade unless I'm mistaken. 
We'd want to modify this such that the existing resources get deleted as well. 
I think modifying the `createDesiredResource` func to return nil should make it such that when the delta is processed, the existing objects should be deleted. Since RHOAI is doing "stable" releases now, this "transition" logic should stay until the next release IMO. After that, we can remove these reconcilers altogether. 
WDYT?

Reply:
that's a good idea, I didn't take into account upgrades, thanks for the heads up.

Reply:
maybe making it returning nil, just add a empty object and remove the function.

Reply:
ok, got it deleted by now.